cohortextractor2/dsl.py

```diff
@@ -161,9 +169,109 @@ class PatientSeries:
     def __init__(self, value: Value):
         self.value = value
 
+    @staticmethod
+    def _get_other(other: Expression) -> Comparator | Value:
```

Comment:
This pipe syntax is new in 3.10. Are we happy to only support that? It would be good if we can, but I suspect at least it means updating some dev environments.

Reply:
That's odd - `black` (I think) autoformatted it, but my local venv is 3.9

Reply:
Actually, it's `pyupgrade` that's changing it